Documentation/ProjectFile/gml/surfaces/surface/element/i_element.md

```diff
@@ -1 +1,3 @@
-\ogs_missing_documentation
+Defines a triangle by its three vertices.
+The face normal of the triangle is determined such that the vertices are
+located counter-clockwise around it (right-hand rule).
```

Comment:
Here are the results of a short conversion with @rinkk:
- The node ordering of the surface triangles haven't to be consistent within a surface.
- The triangles of a surface haven't to be connected necessarily.